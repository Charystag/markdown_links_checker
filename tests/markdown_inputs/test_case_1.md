# Test Case 1: Simple Valid URLs

This test case includes simple, valid HTTP and HTTPS links.

## Example 1: Basic HTTP and HTTPS Links

- [Google](http://www.google.com)
- [GitHub](https://github.com)

## Example 2: Valid Subdomain Links

- [Google Mail](https://mail.google.com)
- [BBC News](https://news.bbc.co.uk)

