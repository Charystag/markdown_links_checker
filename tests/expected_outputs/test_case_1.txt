Checking file: `test_case_1.md'
[0;32mLine: `7'. url: `http://www.google.com' is valid. response_code: `200'[0m
[0;32mLine: `8'. url: `https://github.com' is valid. response_code: `200'[0m
[0;32mLine: `12'. url: `https://mail.google.com' is valid. response_code: `200'[0m
[0;32mLine: `13'. url: `https://news.bbc.co.uk' is valid. response_code: `200'[0m
